tools/c7n_azure/tests/test_function_package.py

```diff
@@ -89,6 +89,17 @@ def test_add_policy(self):
                           u'mode': {u'type': u'azure-event-grid',
                                     u'events': [u'VmWrite']}})
 
+    def test_zipped_files_have_modified_timestamp(self):
+        t = time.gmtime(1577854800)
+        package = AzurePythonPackageArchive()
```

Comment:
I'd add a test when we copy zip archive & ensure old files do have same timestamps.